Sources/MediaPlaybackControlToolbar.swift

```diff
@@ -43,35 +43,32 @@ protocol MediaPlaybackControlToolbarDelegate: class {
     
     lazy var skipBackwardButton: UIButton = {
         var bwdButton = UIButton(type: .system)
-        bwdButton.setImage(UIImage(named: "iconSkipBackward"), for: .normal)
+        bwdButton.setImage(UIImage(named: "iconSkipBack"), for: .normal)
         bwdButton.addTarget(self, action: #selector(skipBackward), for: .touchUpInside)
         bwdButton.tintColor = .white
         return bwdButton
     }()
     
     lazy var skipToPreviousMediaButton: UIButton = {
         var previousMediaButton = UIButton(type: .system)
-        previousMediaButton.setImage(UIImage(named: "iconSkipToPrevious"), for: .normal)
-        previousMediaButton.addTarget(self, action: #selector(skipForward), for: .touchUpInside)
+        previousMediaButton.setImage(UIImage(named: "iconPreviousVideo"), for: .normal)
+        previousMediaButton.addTarget(self, action: #selector(skipToPreviousMedia), for: .touchUpInside)
         previousMediaButton.tintColor = .white
         return previousMediaButton
     }()
     
     lazy var skipToNextMediaButton: UIButton = {
         var nextMediaButton = UIButton(type: .system)
-        nextMediaButton.setImage(UIImage(named: "iconSkipToNext"), for: .normal)
-        nextMediaButton.addTarget(self, action: #selector(skipForward), for: .touchUpInside)
+        nextMediaButton.setImage(UIImage(named: "iconNextVideo"), for: .normal)
+        nextMediaButton.addTarget(self, action: #selector(skipToNextMedia), for: .touchUpInside)
         nextMediaButton.tintColor = .white
         return nextMediaButton
     }()
     
     var isPlaying: Bool {
         didSet {
-            if isPlaying {
-                playPauseButton.setImage(UIImage(named: "iconPause"), for: .normal)
-            } else {
-                playPauseButton.setImage(UIImage(named: "iconPlay"), for: .normal)
-            }
+            let image: UIImage? = isPlaying ? UIImage(named: "iconPauseLarge") : UIImage(named: "iconPlayLarge")
```

Comment:
you don't need to declare UIImage? after image since it's inferred

Reply:
this was more of a coding personal preference where I like to add the type after i declare variables even if it can be inferred but I'll remove it